platform/android/export/export.cpp

```diff
@@ -1896,6 +1896,55 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 		return run_icon;
 	}
 
+	static String get_adb_path() {
+		String exe_ext = "";
+		if (OS::get_singleton()->get_name() == "Windows") {
+			exe_ext = ".exe";
+		}
+		String sdk_path = EditorSettings::get_singleton()->get("export/android/android_sdk_path");
+		return sdk_path.plus_file("platform-tools/adb" + exe_ext);
+	}
+
+	static String get_apksigner_path() {
+		String exe_ext = "";
+		if (OS::get_singleton()->get_name() == "Windows") {
+			exe_ext = ".exe";
```

Comment:
That should be changed to: `exe_ext = ".bat";`

Reply:
Isn't the extension optional on Windows thanks to its `PATHEXT` environment variable?

Reply:
Yeah, but what this code does is actually try to find the tool, so you need the whole path incl. file extension.

Reply:
@Calinou there were a few bugs not too long ago where the lack of the proper extension caused issues on Windows machines (see https://github.com/godotengine/godot/pull/43671). Do you know under what conditions `PATHEXT` takes effect?

@mbrlabs Thanks for checking! I've updated the extension to `.bat`. Can you take another look.

Reply:
@m4gr3d Yes, that fixed it. I also exported an APK & did a quick install it via the editor -> works.